controllers/workspace/condition.go

```diff
@@ -130,3 +132,12 @@ func getConditionIndexInOrder(condType dw.DevWorkspaceConditionType) int {
 	}
 	return -1
 }
+
+func capitalizeMsg(msg string) string {
+	if len(msg) == 0 {
```

Comment:
We usually check for empty strings directly, rather than checking the length of the string. So this can be rewritten as:
```go
if msg == "" {
  return msg
}
```